<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerMainWindow</class>
 <widget class="QMainWindow" name="ServerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>651</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>客户线程</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>621</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>设备线程</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>71</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>设备</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
